### PR TITLE
Ol explainer interactivity

### PR DESCRIPTION
**Pull request by okunator**
This PR adds interactivity to the explainer graph visualization. Namely:
- Tooltip (on hover over nodes and links)
- Path highlighting in the network via graph traversing

### CODE REVIEW

**Comment by tuner** [comment]
What happened to `clin_view.ts`?

**Comment by okunator** [comment]
> 

`clin_view.ts` was unused file so decided to remove it

**Review by tuner — CHANGES_REQUESTED** [review]

File: oncodash-app/src/explainer/graph_utils.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+/**
+ * Depth first travesing of the children links of the explainer graph.
+ * Finds all the nodes/links down stream of a node and also the first parent.
+ * @param {DAGNode} dagNode - Input DAG-node.
+ * @param {string} source - The source node id of the currently active link.
+ * @returns {Map<Link, Node>} - Link-node pairs representing all the nodes/links in a path.
+ */
+export function traverseLinks(
```
[the remark this answers is not in the document]
Comment (okunator) [reply oncodash-app/src/explainer/graph_utils.ts]: fixed at ol-explainer-interactivity

[the remark this answers is not in the document]
Comment (okunator) [reply oncodash-app/src/explainer/graph_utils.ts]: fixed at https://github.com/oncodash/oncodash/pull/47/commits/2ce2e9aace284d95c1f5a1a28971729b87dead9a

**Pull request merged**